lib/tracing-limit/src/lib.rs

```diff
@@ -117,13 +123,13 @@ where
     }
 
     fn on_event(&self, event: &Event<'_>, ctx: Context<'_, S>) {
-        // Visit the event, grabbing the limit if one is defined. If we can't find a rate limit field, or the rate limit
-        // is set as 0, then we let it pass through untouched.
+        // Visit the event, grabbing the limit status if one is defined. If we can't find a rate limit field, or the rate limit
+        // is set as false, then we let it pass through untouched.
         let mut limit_visitor = LimitVisitor::default();
         event.record(&mut limit_visitor);
 
-        let limit = limit_visitor.limit.unwrap_or(0);
-        if limit == 0 {
+        let limit = limit_visitor.limit.unwrap_or(false);
```

Comment:
With a configurable limit, should this feature become opt-out rather than opt-in?

Reply:
Given how most of the internal logs are rate limited, this should become opt-out 👍

Reply:
Ended up keeping it opt-in